products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,178 @@
+Algolia includes analytics, such as ["Most popular searches and results most often returned by the search engine."](https://www.algolia.com/products/analytics/?tab=popular)
+
+### Debugging
+
+The Docker image for the scraper is available on [Docker Hub](https://hub.docker.com/r/algolia/docsearch-scraper). Once we have a config file, we can [run the scraper locally](https://community.algolia.com/docsearch/run-your-own.html#run-the-crawl-from-the-docker-image).
+
+#### Debugging the scraper
+
+#### Debugging the search functionality
+
+We could search for the same term in both the Algolia UI and the landing page to confirm if they both return the same results.
+
+### Caveats
+
+- Access Control: By making the documentation landing page public, it becomes difficult, if not impossible, to allow an unauthenticated user to search documentation from private sources (like a private Github repo). One suggestion is that search results lead to public documentation sources, and we include links to private documentation in the public sources. That way, we can rely on existing access control solutions, while still providing a decent solution. One of the major downsides of that approach is auditing, adding, updating, and maintaining links to/from the public and private sources.
+
+### Security Concerns
+
+If the search input dispatched a request directly to the Algolia API, then there'd be minimal risk of the documentation search negatively affecting other systems. If the request was proxied through our API, then we should be able to rely on any existing DOS protections.
+
+### Privacy Concerns
+
+While there shouldn't be any PII in our public documentation sources, it is possible it exists, and hasn't been discovered yet. Therefore, it is possible that our search results could lead public users to sources that contain PII. If PII is discovered in a repo, we have a [checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/removing-sensitive-data-from-a-repository.md#checklist-for-removing-pii-in-md-file-from-a-documentation-repo) for how to remove it.
+
+#### Removing a record from Algolia
+
+1. Login to Algolia
+1. Navigate to the **Indices** page
+1. Use the search input to find the sensitive record(s)
+1. Click the trash can icon to open the deletion modal
+1. Click the delete button
+
+---
+
+### Open Questions and Risks
+
+---
+
+### Work Estimates
+
+1. Provision and configure data store to house records that will be returned from the Search API. - 1 week
+
+- I estimate one week because I suspect we'll either need to get approval to use Algolia's hosted API or get approval to provision a custom data store.
+
+1. Configure crawler to seed the initial data store. - 2 days
+1. Run the crawler to populate the data store. - 2 days
+1. Build a landing page that will contain a search input that submits requests to the Search API. - 2 days
+1. Deploy the landing page to a publicly available location. - 2 days
+
```

Comment:
Are these estimates still "accurate?" (That is, do you want to update them given any new information we learned and experiments we performed?)